### PR TITLE
Install-DbaInstance - Support SQL Server 2022

### PR DESCRIPTION
**Pull request by andreasjordan**
Followup to #8682

### CODE REVIEW

**Comment by potatoqualitee** [comment]
awesome, thanks andreas! once this is approved and merged, i'll slip it into 1.1.145

File: functions/Install-DbaInstance.ps1 [hunk 1]
```diff
@@ -696,6 +697,9 @@ function Install-DbaInstance {
                     $configNode.SQLTEMPDBFILECOUNT = $cores
                 }
             }
+            if ($canonicVersion -ge '16.0') {
+                $configNode.Remove('X86')
```
Comment (potatoqualitee) [inline functions/Install-DbaInstance.ps1:701]: this may result in a 1 being output, I'll add a null

**Review by nvarscar — APPROVED** [review]
this looks good, thanks @andreasjordan !

**Comment by potatoqualitee** [comment]
Thanks to you both!

**Pull request merged**